### PR TITLE
Various fixes to scaffold and resource generators

### PR DESCRIPTION
**Pull request by bcardarella**

### CODE REVIEW

File: lib/generators/templates/scaffold_controller/controller.rb [hunk 1]
```diff
@@ -1,9 +1,5 @@
-<% if namespaced? -%>
-require_dependency "<%= namespaced_file_path %>/application_controller"
-
-<% end -%>
-<% module_namespacing do -%>
-class <%= controller_class_name %>Controller < ApplicationController
+<% api_version = Rails.application.config.ember.api_version -%>
+class Api::V<%= api_version %>::<%= controller_class_name %>Controller < ApplicationController
```
Comment (abuiles) [inline lib/generators/templates/scaffold_controller/controller.rb:2]: @bcardarella :+1:

File: lib/generators/templates/scaffold_controller/controller.rb [hunk 2]
```diff
@@ -13,7 +9,7 @@ def index
 
   # GET <%= route_url %>/1
   def show
-    render json: <%= "@#{orm_instance.errors}" %>
+    render json: <%= "@#{singular_table_name}" %>
```
Comment (abuiles) [inline lib/generators/templates/scaffold_controller/controller.rb:12]: @bcardarella what about  using AMS as default here? also `respond_to :json` and `respond_with`

Comment (abuiles) [inline lib/generators/templates/scaffold_controller/controller.rb:12]: Actually ignore the AMS part, just `respond_with` and `respond_to`

Comment (bcardarella) [inline lib/generators/templates/scaffold_controller/controller.rb:12]: I'm cool with `respond_to` and `respond_with`

**Pull request merged**